src/panels/lovelace/entity-rows/hui-timer-entity-row.ts

```diff
@@ -40,6 +40,14 @@ class HuiTimerEntityRow extends LitElement {
     this._clearInterval();
   }
 
+  public connectedCallback(): void {
+    super.connectedCallback();
+    if (this._config!.entity) {
```

Comment:
Don't use `!` here. Are you sure that `_config` is available ?

Reply:
There's a check on line 32

Reply:
There is no guarantee that setConfig is called before the element is attached.

Reply:
Okay, I'll update that!

Reply:
Any update on this ?

Reply:
Done